src/Okta.Sdk/Generated/UserSchemaAttributeMaster.Generated.cs

```diff
@@ -17,9 +17,16 @@ namespace Okta.Sdk
     public sealed partial class UserSchemaAttributeMaster : Resource, IUserSchemaAttributeMaster
     {
         /// <inheritdoc/>
-        public string Type 
+        public IList<IUserSchemaAttributeMasterPriority> Priority 
         {
-            get => GetStringProperty("type");
+            get => GetArrayProperty<IUserSchemaAttributeMasterPriority>("priority");
+            set => this["priority"] = value;
+        }
+        
+        /// <inheritdoc/>
+        public UserSchemaAttributeMasterType Type 
```

Comment:
Let's add details about this change, explaining why this is not technically a breaking change. Let's add a small UT.
string profile.Type;